test/resources/formula-math-test.ts

```diff
@@ -75,6 +75,7 @@ describe('formulas and mathml', () => {
 
   test('should properly identify and convert latex formulas', () => {
     let s = process('<body><formula>$$\\frac{2}{3}$$</formula></body>');
+    console.log(s);
```

Comment:
Is this console log intended to be here?

Reply:
Whoops, no.